### ISSUE TITLE
Add prometheus alert for read amplification issues

### ISSUE THREAD

**Issue by glennfawcett** [cockroachdb/cockroach#56129]
**Is your feature request related to a problem? Please describe.**
Ran into an issue with runaway amplification

**Describe the solution you'd like**
New Prometheus alert to be created to notify of high read amplification sooner.
<img width="1308" alt="Screen Shot 2020-10-23 at 9 16 44 AM" src="https://user-images.githubusercontent.com/5796677/97635851-4c235d00-19f5-11eb-9754-df52d2959600.png">
<img width="1308" alt="Screen Shot 2020-10-23 at 9 16 44 AM" src="https://user-images.githubusercontent.com/5796677/97635885-5f362d00-19f5-11eb-88bc-dc7498f18cec.png">
<img width="1034" alt="Screen Shot 2020-10-29 at 2 46 25 PM" src="https://user-images.githubusercontent.com/5796677/97635999-90166200-19f5-11eb-9329-aa0153a9b8d0.png">
![Screen Shot 2020-10-29 at 2 46 25 PM](https://user-images.githubusercontent.com/5796677/97636091-ba681f80-19f5-11eb-8c25-47c69aae9788.jpg)

Basically, update:

- [alert-rules.yaml](https://github.com/cockroachdb/cockroach/blame/master/cloud/kubernetes/prometheus/alert-rules.yaml)

**Comment by tbg** [comment]
FYI the L2 grafana dashboard[^1] has this alert. (IO Admission Control) 

<img width="961" alt="image" src="https://user-images.githubusercontent.com/5076964/234689221-6ce48d5f-2efd-4bff-8ac6-3b0da21016ac.png">

which basically engages (when IO admission control is enabled and) the LSM is inverting.

This dashboard is something we're using for an internal testcluster. It's not a standard dashboard but I see it as an incubator ideas in which will be picked up over time.

[^1]: https://grafana.testeng.crdb.io/d/CAvWxELVz/l2-drill-down?orgId=1&from=now-12h&to=now
